concepts/observability-configurable-logging/02-agents.md

```diff
@@ -0,0 +1,56 @@
+It is quite new but already has a big community behind as it is covering the community of OpenCensus and OpenTracing. With that the ecosystem is already good enough and the majority of vendor-specific backends are supported (seeing the available exporters on the contrib repo) being based on a new vendor-neutral protocol.
+It is most probably the only real vendor-neutral project covering really all three aspects of logs, traces and metrics.
+Bigger companies are actively planning to use it at least for the trace and metrics aspects. Here, the configuration specification is still not in a stable state. For logging, the whole feature is still in a beta stadium.
+See also [more detailed here](../observability-opentelemetry/README.md)
+
+**Logs - Fluent-Bit**
+It is the traditional log collector used in Kyma already, being part of CNCF and widely adopted on K8S. It is vendor-neutral and supports a variety of generic outputs being capable of integrating with quite some backends. If a backend is not supported, you can use the fluent specific Forward protocol to connect a FluentD providing a even more richer ecosystem. It will scale-out perfect by running on every node with a minimalistic resource footprint.
+
+**Logs - FluentD**
+Written in Ruby and being more resource extensive then fluent-bit, FluentD presents a valid alternative to fluent-bit. However, it is usually recommended as the central application not running on every node but doing the central aggregation and shipment to the backend.
+
+**Logs - rsyslog**
+It also supports a variety of inputs and outputs. Here mainly the built-in kubernetes support is missing.
+
+**Logs - Logstash**
+Popular project with a rich ecosystem with built-in kubernetes support. Developed under the Elastic umbrella and a clear addiction to use ElasticSearch as a backend.
+
+**Logs - Promtail**
+Is not vendor-neutral on the output side, mainly designed for the Loki backend
+
+**Logs - Flume**
+Seems to have no active community anymore
+
+**Traces - OpenCensus/Jaeger Agent**
+Both agents are deprecated in favor of using otel-collector.
+
+**Metrics - Prometheus**
+Prometheus is a collector and backend at the same time. While you could reduce the backend settings to a minimum and mainly use the metrics forwarding to act as a collector/forwarder, the solution will still be not flexible. Only prometheus compatible backends will be supported.
+
+**Metrics - Telegraf**
+Has a broad community, supports various input/outputs and supports kubernetes
+## Conclusion
+
+Seeing the wide adoption of OpenTelemetry and the grow of the Community, seeing that it will cover all three data kind in a consistent way, seeing it fully vendor-neutral, seeing it written in a resource-efficient and scalable way, then the `otel-collector` is the way to go. It covers all requirements and brings the consistency on top.
```

Comment:
```suggestion
The `otel-collector` is the way to go because of the following reasons:
- OpenTelemetry is widely adopted and the community is growing.
- It covers all three kinds of telemetry data in a consistent way.
- It's fully vendor-neutral. 
- It's written in a resource-efficient and scalable way.
The `otel-collector` covers all requirements and on top, it brings consistency.
```